packages/device-client/src/lib/types/oath.types.ts

```diff
@@ -5,11 +5,13 @@
  * of the MIT license. See the LICENSE file for details.
  */
 export type OathDevice = {
-  id: string;
+  _id: string;
+  deviceManagementStatus: boolean;
   deviceName: string;
   uuid: string;
-  createdDate: Date;
-  lastAccessDate: Date;
+  createdDate: number;
+  lastAccessDate: number;
+  _rev: string;
```

Comment:
This reminds me of a MongoDB document. I wonder if they are using a NoSQL data store for storying this info.